templates/zerver/help/viewing-messages-from-stream.md

```diff
@@ -0,0 +1,17 @@
+# View messages from a stream
+
+![home view](/static/images/help/home-view.png)
+
+In the home view, you'll see messages from all streams and topics. However, you may only want to see messages from a single stream. This is also called 'narrowing' to a stream.
+
+To view messages from a stream, select the stream from the stream list on the left.
+
+![narrowed view](/static/images/help/narrowed-view.png)
+
+Only messages from the selected stream will be displayed. 
+
+In the stream list, the stream whose messages are currently being viewed will be highlighted in blue, and the topics in that stream will be displayed below the stream name.
+
+An alternative way to view messages from only one stream is to type **stream:** in the search bar at the top, and then enter the name of the stream whose messages you want to view. Autocomplete options to narrow to a stream will appear as you type.
```

Comment:
This should mention that it's possible to view public streams you're not subscribed to, by just typing the stream name in the search bar.  (We're also planning to have a "preview" button on the /#subscriptions page, but that's not done yet).